custom-routes-vis/src/main/webapp/script.js

```diff
@@ -209,41 +210,26 @@ function generateRoutes() {
  * from the selected API.
  * @param {!Object} routeJson JSON object containing all information of the target
  * route.
- * @param {boolean} directionsApi Whether the route is obtained by the Directions
+ * @param {boolean} usedDirectionsApi Whether the route is obtained by the Directions
```

Comment:
Could you rename it to `isDirectionsApiRoute`?

Reply:
ok